app/views/guests/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @guests.name %></h1>
+<p>Suite: <% @guests.rooms.each do |room| %></p>
+  <p><%= room.suite %></p>
+    <% end %>
+<p>Rate: <% @guests.rooms.each do |room| %></p>
+  <p><%= room.rate %></p>
+    <% end %>
+<p>Hotel name: <% @guests.rooms.each do |room| %></p>
+  <p><%= room.hotel_id %></p>
```

Comment:
It looks like this part tripped you up on US 1. We are looking for the name of the hotel and it's actually easier than it seems -> because of the AR associations we've set up (`room` belongs to a hotel), we can call `room.hotel.name` and get the name of the hotel. Set up a pry in your controller or view and try calling this. It's easier than you're expecting!